contrib/scripts/kind.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# Sample usages
+# ./contrib/scripts/kind.sh
+# IPv6=1 ./contrib/scripts/kind.sh
+# NO_BUILD=1 ./contrib/scripts/kind.sh
+# NO_PROVISION=1 ./contrib/scripts/kind.sh
+
+# Enable IPv6 mode. By default using IPv4
+export 'IPv6'="${IPv6:0}"
```

Comment:
Not a fan of the mixed case for the variable (I understand we're already using this name in other scripts, though). By the way, you DO mix up `IPv6` here and `IPV6` (all caps) later in the script. Please fix!